packages/Radio/src/components/Group/Group.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+
+const propTypes = {
+  /** aria-labelledby prop on the containing group element */
+  a11yLabelledByText: PropTypes.string,
+
+  /** Can deselect any radio */
+  canDeselect: PropTypes.bool,
+
+  /** Function used to evaluate which radio is selected by default. */
+  defaultCheck: PropTypes.func,
```

Comment:
🤔 ...

Reply:
This might be the most significant aspect I would like to challenge... This seems a bit awkward, and I like the way the API for the `<ListBox>` handles this better, though I'm not so familiar with the implementation details.  

This prop could be omitted in favour of adding an `isSelected` (or `defaultSelected`) prop directly on the `<Radio>` item that is selected. Consumers will likely map through an array in their application data to generate those `<Radios>` and so adding `defaultSelected` should be pretty straightforward.  But then the `<Radio>` component is going to have to set that in the state for the `RadioGroup` somehow... 🤔 

Does this approach make sense to you, @nahumzs?

Reply:
I see in the list box option component we have both `defaultIsSelected` and `isSelected` props available, which i find a bit confusing. @nahumzs What was the reason again for including both? Do you think that would be beneficial to introduce both here?

Reply:
For the Listbox the reason is that sometimes you don't want to handle the isSelected prop for the component you want it to be uncontrolled and only listening for the onChange event (defaultIsSelected); but also there are cases where there is a need for the consumer to fully control the isSelected prop (control component).

`Btw` Most of react components are controlled and uncontrolled components like the `<input />` where you can use defaultValue or value. 

Handling both cases on the listbox required to evaluate each item and useEffect to listening for changings to synchronize the internal state. (because listbox has an useReducer)

In contrast in the listBoxBrowser I'm using a function instead of adding a prop to each element, the idea of doing that was as inspiration after talking about this problem with Andrei. 

Extracting which element is selected and which no to a function make the component more flexible, now we don't have to care about how to store the values and the consumer can use any kind of data structure to store the selected values.

It is more flexible in my experience and removes complexity from the author and the consumer. 

So for this specific component, might be easier for the consumer to use `isSelected`, `defaultSelected` to each individual checkbox. For that I'm sorry because I suggested using the function approach.  

But in a more complex scenario is easier to go with the function pattern. 

Let me know if you need any assistance to make this change and I can help you. Sorry again to suggested the function approach at the beginning.